# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webrtc/libwebrtcglue/MediaConduitInterface.h
# Commit: a1c16b86d3a7
# Full Hash: a1c16b86d3a7f328a690c00f86b837a8a67a656c
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2021-11-01 18:17:33
# Regressor Bug: 1654112
# File Overlap Count: 5
# Description:
#   Bug 1654112 - Remove RtcpEventObserver. r=ng
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D102287
# ==============================================================================

diff -r c7dbdd82e8e8 -r a1c16b86d3a7 dom/media/webrtc/libwebrtcglue/MediaConduitInterface.h
--- a/dom/media/webrtc/libwebrtcglue/MediaConduitInterface.h	Mon Feb 01 09:28:23 2021 +0100
+++ b/dom/media/webrtc/libwebrtcglue/MediaConduitInterface.h	Mon Jan 18 16:41:26 2021 +0100
@@ -11,7 +11,6 @@
 #include "mozilla/RefPtr.h"
 #include "mozilla/RefCounted.h"
 #include "mozilla/UniquePtr.h"
-#include "RtcpEventObserver.h"
 #include "CodecConfig.h"
 #include "VideoTypes.h"
 #include "jsapi/PeerConnectionCtx.h"
@@ -241,7 +240,6 @@
   virtual Maybe<RefPtr<VideoSessionConduit>> AsVideoSessionConduit() = 0;
 
   virtual webrtc::Call::Stats GetCallStats() const = 0;
-  virtual void SetRtcpEventObserver(RtcpEventObserver* observer) = 0;
 
   NS_INLINE_DECL_THREADSAFE_REFCOUNTING(MediaSessionConduit)
 };